### ISSUE TITLE
Don't join World.

### ISSUE THREAD

**Issue by hiro1234omochi** [rtm516/MCXboxBroadcast#22]
Cannot join World

It appears in my friends list, but I get a message like "could not join world" and cannot join.
According to my verification, I can enter correctly on the local network.

And one question, what is the difference between the standalone version and the extension version?

**Comment by PizzaThatCodes** [comment]
To answer your question the main difference between the two would be the extension is made to run along side your server (inside the GeyserMC Plugin extension folder) which means you don't need a side program to go with your server to run it, while the standalone is a separate program overall. And currently speaking the standalone has AutoFriend, while the geyser extension has a pull request for it, but its not currently accepted.

TLDR: extension doesn't need to be ran as a side program & doesn't (yet) have AutoFriend. While Standalone is a separate program but currently has AutoFriend.

**Comment by PizzaThatCodes** [comment]
I do have a few questions for you as well to help me figure out what the issue is.

1. Which version do you use (standalone or extension)?
2. Did you make sure the GeyserMC config.yml file has the proper port linked up? (And maybe the IP)
3. If your using the extension have you tried changing the IP and port from 'auto' to the actual IP and port?
4. Have you tried to use Windows or PE to join the server? What I mean by that is adding the IP & port to the server list and making sure it does work.
5. If number 4 works, have you made sure that anyone can get the server info (you can test your server connection here: https://mcsrvstat.us) If that turns up negative then it could be something with the IP and port, firewall, and/or port forwarding with your server.

**Comment by rtm516** [comment]
This is likely down to using the default `auto` setting which will pull your external facing IP address which if you havent port forwarded connections to fail. As mentioned by @PizzaThatCodes try setting it to the actual IP and port of the server on your local network.

**Comment by hiro1234omochi** [comment]
After entering the actual address from auto, I was able to do it. Thank you for your response.
